src/res/flame.html

```diff
@@ -61,14 +61,35 @@ <h1>/*title:*/</h1>
 		[0xcce880, 20, 20, 20],
```

Comment:
Please add a sample file to `.assets/html/flamegraph-diff.html` - and perhaps reference it from the docs.